xAPI.md

```diff
@@ -2749,12 +2725,12 @@ expected (a MUST requirement on the LRS) to keep precision to at least milliseco
 seconds).  
 
 ###### Requirements
-* A timestamp MUST be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).
-* A timestamp SHOULD* include the time zone.
-* If the timestamp includes a time zone, the LRS MAY be return the timestamp using a different timezone to the one originally used in the statement so long as the point in time referenced is not affected. The LRS SHOULD* return 
-the timestamp in UTC timezone. 
-* A timestamp MAY be truncated or rounded to a precision of at least 3 decimal digits for seconds (millisecond precision MUST be preserved). 
-* An LRS MUST NOT reject a timestamp for being from the future, to prevent issues due to clock errors.
+* A Timestamp MUST be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).
+* A Timestamp SHOULD* include the time zone.
+* If the Timestamp includes a time zone, the LRS MAY be return the Timestamp using a different timezone to the one originally used in the statement so long as the point in time referenced is not affected. 
+* The LRS SHOULD* return the Timestamp in UTC timezone. 
+* A Timestamp MAY be truncated or rounded to a precision of at least 3 decimal digits for seconds (millisecond precision MUST be preserved). 
+* An LRS MUST NOT reject a Timestamp for being from the future, to prevent issues due to clock errors.
```

Comment:
* This probably has to be a SHOULD NOT* since this line about the future wasn't there in 1.0.0
* It seems reasonable and easiest to just apply this to the data type, it's only really important for statement timestamp and stored time, but it doesn't seem particularly harmful to require it elsewhere. Is it really necessary to allow an LRS to return an error if a "since" timestamp is in the future, rather than just returning (presumably) no results?